rozwiazania_tesktowe.txt

```diff
@@ -0,0 +1,13 @@
+3.2.4 UML diagram:
+https://drive.google.com/file/d/1FOCU3PVdsIbdJp1pdQPjLzPb-In64F7r/view?usp=sharing
```

Comment:
Attribute drawPanelController does not need to be listed (it is represented by an aggregation).

Reply:
But it is just not obligatory, yet is not a mistake?

Reply:
I did not considered it a mistake.